lib/Bridge/TolerantParser/Reflection/ReflectionMethod.php

```diff
@@ -130,15 +138,21 @@ public function visibility(): Visibility
 
     /**
      * If type not explicitly set, try and infer it from the docblock.
+     *
+     * @deprecated Use inferredReturnTypes()
      */
     public function inferredReturnType(): Type
```

Comment:
Should probably add a `best()` method which returns the first or an Unknown type, as often a decision will need to be made/guessed.